components/favourites.js

```diff
@@ -97,42 +86,56 @@ export class Favourites extends Component {
         />
-        <div className={whiteBanner}>
-          <Container>
-            <Grid container spacing={24}>
-              <Grid item xs={12} className={right}>
-                <HeaderLink
-                  href={this.props.printUrl}
-                  target="print_page"
-                  id="printButton"
-                >
-                  <Print /> {t("Print")}
-                </HeaderLink>
-              </Grid>
-            </Grid>
-          </Container>
-        </div>
-
         <Container id="favourites">
-          <Grid container spacing={24}>
-            <Grid item xs={12} className={topMatter}>
-              <Header className={"BenefitsCounter"} size="xl" headingLevel="h1">
-                {t("favourites.saved_benefits", {
-                  x: filteredBenefits.length
-                })}
-              </Header>
+          <Grid container spacing={32}>
+            <Grid item lg={4} md={4} sm={5} xs={12}>
+              <Grid container spacing={16} className={favouritesLink}>
+                <Grid item xs={12}>
+                  <HeaderButton
+                    id="nextSteps"
+                    onClick={() => {
+                      window.scrollTo({
+                        top: this.nextStepsRef.current.offsetTop,
+                        behavior: "smooth"
+                      });
+                    }}
+                  >
+                    <AssignmentTurnedIn />
+                    {t("nextSteps.whats_next")}
+                  </HeaderButton>
+                </Grid>
+              </Grid>
+              <ShareBox
+                t={t}
+                printUrl={this.props.printUrl}
+                url={url}
+                share={false}
+              />
             </Grid>
-            <Grid item md={8} xs={12}>
-              {this.state.showDisabledCookieBanner ? (
-                <DisabledCookiesBanner
-                  t={t}
-                  onClose={() =>
-                    this.setState({ showDisabledCookieBanner: false })
-                  }
-                />
-              ) : null}
+            <Grid item lg={8} md={8} sm={7} xs={12}>
+              <Grid container spacing={16}>
+                <Grid item xs={12}>
+                  {this.state.showDisabledCookieBanner ? (
+                    <DisabledCookiesBanner
+                      t={t}
+                      onClose={() =>
+                        this.setState({ showDisabledCookieBanner: false })
+                      }
+                    />
+                  ) : null}
+                </Grid>
+              </Grid>
 
               <Grid container spacing={24}>
+                <Header
+                  className={"BenefitsCounter"}
+                  size="xl"
+                  headingLevel="h1"
+                >
+                  {t("favourites.saved_benefits", {
+                    x: filteredBenefits.length
+                  })}
+                </Header>
```

Comment:
`<Body>{t("B3.check eligibility")}</Body>` to add the "Please read the eligibility criteria" text under the heading.